packets1/packets1.go

```diff
@@ -83,9 +83,11 @@ func ReadPacket(r io.Reader) (pkt pkts.Packet, err error) {
 	h.Unpack(packet)
```

Comment:
When studying the code I noted that `Header.Unpack` can also return an error (when packet length < 2), but this is not checked here. I guess it should be? Or is it covered by the requirement that `r` must be a packet reader?

Reply:
You are right, thanks. I will fix it too.

Reply:
Could you also add a test for that?